.eslintrc

```diff
@@ -2,25 +2,17 @@ env:
   node: true
   mocha: true
 
+extends: 'eslint:recommended'
 rules:
-  comma-style: [2, "last"]
-  default-case: 2
-  func-style: [2, "declaration"]
-  no-floating-decimal: 2
-  no-nested-ternary: 2
-  no-undefined: 2
-  radix: 2
-  space-after-keywords: [2, "always"]
-  space-before-blocks: 2
-  spaced-line-comment: [2, "always", { exceptions: ["-"]}]
-  valid-jsdoc: [2, { prefer: { "return": "returns"}}]
-  wrap-iife: 2
-  quotes: "single"
-  strict: false
-  new-cap: false
-  no-multi-spaces: false
-  curly: true
-  guard-for-in: false
-  no-underscore-dangle: false
-  no-new: false
-  no-wrap-func: false
```

Comment:
Are all the rules above carried over below? Looks like there's way fewer lines. Are they all included in `eslint:recommended`?